api/IntersectionObserver.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "IntersectionObserver": {
+        "__compat": {
+          "basic_support": {
+            "support": {
+              "chrome": {
+                "version_added": "51"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "firefox": [
```

Comment:
I would have expected this the other way around. Basically there are two ranges:
 * 53-55: support behind a pref
 * 55+: supported

I would express that as:

```json
"firefox": [
  {
    "version_added": "53",
    "version_removed": "55",
    "flag": {
      "type": "preference",
      "name": "dom.IntersectionObserver.enabled",
      "value_to_set": "true"
    }
  },
  {
    "version_added": "55",
   }
]
```

It's not clear to me what details the bug reference would bring to the reader here. I wouldn't add it (and it is also not actually the bug that enabled `IntersectionObserver`).